.github/workflows/build_release.yml

```diff
@@ -16,7 +16,19 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, macos-14, windows-latest]
+        include:
+          - os: ubuntu-20.04
+            arch: x86_64
+            name: amd64-ubuntu-20.04
+          - os: ubuntu-24.04-arm64
+            arch: aarch64
+            name: arm64-ubuntu-24.04
+          - os: macos-14
+            arch: aarch64
+            name: arm64-macos-14
+          - os: windows-latest
+            arch: x86_64
+            name: amd64-windows-latest
```

Comment:
Can we just call this one `windows` for now?  (At least until we support more than one windows arch).